tests/console/journalctl.pm

```diff
@@ -258,12 +284,17 @@ sub cleanup {
 }
 
 sub post_fail_hook {
+
     select_console 'log-console';
-    shift->save_and_upload_log('journalctl', '/tmp/journalctl.txt', {screenshot => 1});
+    script_run 'cp -a /var/log/journal /var/log/journal-backup';
+    script_run 'tar Jcvf journal.tar.xz /var/log/journal/';
+    script_run 'tar Jcvf journal-backup.tar.xz /var/log/journal-backup';
```

Comment:
Why do you store it twice? Isn't it the same content? or does `cp -a` something special?

Reply:
That was requested by Frank (systemd dev) during investigation of https://bugzilla.suse.com/show_bug.cgi?id=1182802. VMware machine had a time shift 25+ minutes, so whenever we have rebooted the guest, it took sometime for `chronyd` to adjust the system time of the guest according to remote timeservers. There is a catch thought that the issue can be masked when chronyd shifts the time and journald performs a log rotation. So he asked to create an archive of the whole directory whenever the issue described in the bug occurs and openQA detects it. Although it was a host setup problem there is a pitfall that logs can be rewritten in case another reboot occurs shortly after `chronyd` shifts the system time of the guest. Frank suggested to make journald somehow aware of this situation, but we will need to revert the changes on VMware host ( make significant time delay again ) and collect additional system logs.

Reply:
thx for explanation. I think I somehow understand that thing about reboot and timesync.
But what difference we expect from these two archives, I don't, or do we think there might be a timeshift just between these commands.

Reply:
Yup, or that the logs will get rotated meanwhile. So there will be a new file entry

Reply:
hm...but maybe I can change the order or add `sleep`...

```
script_run 'tar Jcvf journal.tar.xz /var/log/journal/';
script_run 'cp -a /var/log/journal /var/log/journal-backup';
script_run 'tar Jcvf journal-backup.tar.xz /var/log/journal-backup';
```

Reply:
I think a sleep would make sense